### ISSUE TITLE
Allow users to manage 'read' status manually

### ISSUE THREAD

**Issue by FP1996** [CTemplar/webclient#1281]
### Expected behavior

Not everyone uses email the same way.
Email is historically described very well as a TODO list that other people can append things to. And many people use it as such.

One very obvious way of doing this is that an email is not marked "read" until it has been handled.

To assist in this way of working it would be really nice if a user could set a config option to not mark emails, as is done now, by the mere action of clicking on them. Instead such users would use the "mark mail read" menu item (hopefully also having a keyboard shortcut).

So, in short,

[ ] Allow emails to not be marked read automatically ever. (only manually)
[ ] Create a mailbox-wide option to enable this behavior.

**Comment by The-Hidden-Hand** [comment]
Accepted for development. We are going to include it in user settings (we'll continue marking it as read by default).

**Comment by nitishkansal** [comment]
Backend is done, setting is added `auto_read` and updated auto read feature as well to work with this setting.

**Comment by The-Hidden-Hand** [comment]
@superdev0714 
Where is the user setting? Now the emails are never marked as read and the setting is non existing. The function is broken, isn't it? Also, the issue doesn't follow the ticket. Now the "go back after marking unread" feature is always disabled, which is not expected.

**Comment by superdev0714** [comment]
@The-Hidden-Hand I will add option UI to check auto or manual read way in general setting.

**Comment by The-Hidden-Hand** [comment]
@superdev0714 

Now:

- When I enter any unread message it will stay unread unless I do manually
- Marking as unread will not redirect me back in any case. It should redirect back only when I have `autoread on`, but not when `autoread off` because in that case I can read any message and it will still be "unread".
- There is no setting to control this function
